packages/ui/src/EditorUI/__tests__/index.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import EditorUI from '../';
+
+describe('EditorUI', () => {
+  const defaultProps = {
+    stickyNotes: {
+      shouldStickToTop: false,
+      shouldStickToBottom: false,
+      rightOffset: 0,
+    },
+    hideEditorSidebar: false,
+  };
+
+  it('renders correctly with default properties', () => {
+    const component = shallow(<EditorUI {...defaultProps} />);
+
+    expect(component).toMatchSnapshot();
+  });
+
+  it('renders correctly with option to hide the editor sidebar', () => {
+    const component = shallow(<EditorUI {...defaultProps} hideEditorSidebar />);
+
+    expect(component.find('DisplayModeToggle')).toHaveLength(0);
+  });
+});
```

Comment:
nice! we should definitly start to do more component tests

Reply:
There are a lot of approaches for tests in React. There are people that prefer to test on a big scale. Usually, I do in a component scale, but you can use the best one for your project.